core/javatests/com/google/instrumentation/stats/TagTest.java

```diff
@@ -31,6 +31,29 @@ public void testTagTest() {
     assertThat(tag.getValue()).isEqualTo(V1);
   }
 
+  @Test
+  public void testEquals() {
+    assertThat(Tag.create(K1, V1)).isEqualTo(Tag.create(K1, V1));
+    assertThat(Tag.create(K1, V1)).isNotEqualTo(Tag.create(K2, V1));
+    assertThat(Tag.create(K1, V1)).isNotEqualTo(Tag.create(K1, V2));
```

Comment:
I thought about that but I'm mostly checking isNotEqualsTo() so I don't think it adds anything.